apps/collection/behaviours/confirm.js

```diff
@@ -0,0 +1,35 @@
+'use strict';
+
+var _ = require('underscore');
+
+function getReason(req) {
+  var reasons = [
+    'which-post-office',
+    'under-age',
+    'non-identity',
+    'others-identity',
+    'passport-family',
+    'passport-lost',
+    'no-brp',
+    'other'
+  ];
+  var reason = {};
+
+  if (_.includes(reasons, req.form.values['reason-radio'])) {
```

Comment:
I could, but if we go through an app that's well over a year old and decide to make all possible code improvements then this will never get merged. 

Can we call it out of scope for now?